_posts/2020-08-18-scaffolding-single-page-applications-with-cdk.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: Scaffolding your AWS single page application infrastructure with CDK
+date: 2020-08-24
+tags: aws cdk static-site react angular cloud-development-kit infrastructure-as-code spa single-page-application
+author: Tim Veletta
+---
+
+Many of client jobs we do at Mechanical Rock involve the creation of single page applications (SPAs) so it often helps to have a simple, repeatable process for quickly scaffolding one. The AWS Cloud Development Kit (CDK) is a relatively new framework for defining your cloud infrastructure as code to be provisioned with CloudFormation. CDK allows developers of SPAs to use the tools they are already familiar with, in this case TypeScript (or JavaScript), to define a template for their infrastructure that can be reused from one project to the next.
```

Comment:
`Many of client jobs`...isn't words.
consider:
`Many projects at Mechanical Rock`

Reply:
🤦